doc/user/zebra.rst

```diff
@@ -580,6 +580,48 @@ nexthops are chosen to forward packets on.  Currently the Linux kernel
 has a ``fib_multipath_hash_policy`` sysctl which dictates how the hashing
 algorithm is used to forward packets.
 
+.. _zebra-svd:
+
+Single Vxlan Device Support
+===========================
+
+FRR supports a new way of configuring VLAN-to-VNI mappings for EVPN-VXLAN,
```

Comment:
I think I'd take out "new way" here ... at some point this is going to be the "old way," which might make the code confusing :-)

Reply:
done